jobs/otel-collector/templates/bpm.yml.erb

```diff
@@ -6,8 +6,15 @@
     }
 
     memory_mib = p('limits.memory_mib')
-    process['limits'] = { 'memory' => "#{memory_mib}MiB" }
-    process['env'] = { 'GOMEMLIMIT' => "#{(memory_mib.to_i * 0.80).floor}MiB" }
+    cpu = p('limits.cpu')
+    process['limits'] = { 
+                          'memory' => "#{memory_mib}MiB",
+                          'cpu' => cpu.to_i
+                        }
+    process['env'] = { 
```

Comment:
Can you align these indentations with the established pattern in the file

Reply:
Done.